src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -0,0 +1,443 @@
+| DispersionConstant                  | RECOMMENDED           | [number][]    | External dispersion time constant resulting from tubing in default unit seconds.                                                                                                                                                                              |
+| Haematocrit                         | RECOMMENDED           | [number][]    | Measured haematocrit, meaning the volume of erythrocytes divided by the volume of whole blood.                                                                                                                                                                |
+| BloodDensity                        | RECOMMENDED           | [number][]    | Measured blood density. Unit of blood density should be in g/mL.                                                                                                                                                                                              |
+
+### Time
+
+If manual blood samples, for example, using a COBRA counter are available,
+then time information as detailed below should be added to the `*recording-manual_blood.tsv`
+and the values below should be added to the `*recording-manual_blood.json`.
+If blood samples using an autosampler, for example, using an Allogg autosampler are available,
+then time information as detailed below should be added to the `*recording-autosampler_blood.tsv`
+and the values below should be added to the `*recording-autosampler_blood.json`.
+
+| **Column name** | **Description**                                                              | **Units**                             |
+| --------------- | ---------------------------------------------------------------------------- | ------------------------------------- |
+| time            | Time in relation to `TimeZero` defined by the `*_pet.json`. (for example, 5) | Unit of time steps (for example, "s") |
+
+### Radioactivity in plasma
+
+This section may be omitted if plasma measurements of radioactivity were not made.
+It contains information regarding sampled plasma data.
+If plasma measurements are available by manual sampling, they should be added to the `*recording-blood_discrete.tsv`
+and the values below should be added to the `*recording-blood_discrete.json`.
+If plasma measurements are available by an autosampler, they should be added to the `*recording-blood_continous.tsv`
+and the values below should be added to the `*recording-blood_continous.json`.
+
+| **Column name**      | **Description**         | **Units**                                         |
+| -------------------- | ----------------------- | ------------------------------------------------- |
+| plasma_radioactivity | Radioactivity in plasma | Unit of parent fraction (for example, "unitless") |
```

Comment:
it is unclear to me how I would figure out the units of the numbers that are given in this table. The table would for example look like this 

```
plasma_radioactivity
32.4
54.3
21.3
13.4
1.2
...
```

which field in which JSON explains what these numbers mean?

Reply:
The units are given in the corresponding *.json file. https://github.com/mnoergaard/bids-specification/blob/bep-009/src/04-modality-specific-files/09-positron-emission-tomography.md#example-_recording-manual_bloodjson

Reply:
Thanks for catching this, there was actually a wrong statement under unit. @mnoergaard has fixed it.

Reply:
Thanks for catching this, there was actually a wrong statement under unit. @mnoergaard has fixed it.